### ISSUE TITLE
Pruning does not work in scheduled tasks

### ISSUE THREAD

**Issue by lldsolitude** [OpenMediaVault-Plugin-Developers/openmediavault-compose#27]
Hi, 

I set up a scheduled task in compose plugin with both update and prune enabled, but after the scheduled task executed, the images were updated but dangling images were not pruned as expected. I tried setting up a task with only prune selected but an error popped up saying "You must select at least one action type".

As a previous issue pointed out I could just set up a general scheduled task to prune dangling images, but now that the function is already in the plugin it would make sense for it to work?

I'm running omv 7.0.5 with openmediavault-compose 7.1.3

Thanks!

**Comment by ryecoaaron** [comment]
> now that the function is already in the plugin it would make sense for it to work?

Of course.  I will look at it.

**Comment by ryecoaaron** [comment]
I have fixed the issue where you can't select only prune for a job.

> I set up a scheduled task in compose plugin with both update and prune enabled, but after the scheduled task executed, the images were updated but dangling images were not pruned as expected.

The plugin is just running "docker image prune -f".  If that doesn't prune an image, docker doesn't think it can be pruned.  Can you post the output of the following:

cat /etc/cron.d/omv-compose-prune

**Comment by ryecoaaron** [comment]
> Is this the intended behavior?

Kind of.  Even though the update script gets kicked off at the same time as the backup script, the update script waits for the backup script to complete.  When I added the prune, I figured it would be scheduled by itself but it doesn't have to be.  It just means there might be an extra image around that would be pruned the next time it runs.  I would have to write a wrapper script around the docker prune command to wait for the update script to finish.
